### DISCUSSION TITLE
Error updating Next.js to versions above 13.0

### DISCUSSION THREAD

**Discussion by ghost** [vercel/next.js#45661] [Help] [answered]
**next-auth: 4.19 works fine in next: 13.0**
**but updating Next.js to versions above ^13.0 throws this error:**

![Ativo 3](https://user-images.githubusercontent.com/67167087/217079338-cf480675-7ca6-4a60-8b94-244f69e70576.png)
![error1](https://user-images.githubusercontent.com/67167087/217074869-5ba66a9d-27f3-4c59-b51f-222aede3f429.png)
![error2](https://user-images.githubusercontent.com/67167087/217074890-ab4c665a-b35a-4a32-b3bd-6f5fcd8dfa4b.png)
I have no idea how to solve it, please help me I'm at your disposal.

**Comment by ciruz** [comment, marked as answer]
Did you tried any other next.js 13 versions too? For example ``v13.1.5``, ``v13.1.1``, ``v13.1.0``, etc.?

Maybe you can find the last working ``v13`` version for you. We can dig further if we find the last working version.

**Reply by ghost** [reply]
The issue starts at `v13.0.7` exactly.

When runs this code: `Route.replace('/', undefined, { shallow: true })`

![Ativo 1](https://user-images.githubusercontent.com/67167087/217287958-db15f80c-e6a3-4a91-b726-8e1334d8b61e.png)

**Reply by ciruz** [reply]
do you have a little code example repo?

**Reply by ghost** [reply]
Changing `replace('/', undefined, { shallow: true })` to `replace('/', undefined, { scroll: false })` solved my problem, thanks.
